howso/engine/trainee.py

```diff
@@ -1108,12 +1060,12 @@ def analyze(
         analysis_sub_model_size: Optional[int] = None,
         analyze_level: Optional[int] = None,
         p_values: Optional[List[float]] = None,
-        targeted_model: Optional[str] = None,
+        targeted_model: Optional[TargetedModel] = None,
```

Comment:
I wonder if `None` should just be one of the possible `TargetedModel` types?

Reply:
That would certainly simplify things but I think it obfuscates the `Optional` nature of the parameter and implies that `None` does something in particular (rather than just being unsupplied), so I prefer the way it is.